Dangerfile.swift

```diff
@@ -20,3 +21,13 @@ var bigPRThreshold = 1000;
 if (danger.github.pullRequest.additions! + danger.github.pullRequest.deletions! > bigPRThreshold) {
     warn("> Pull Request size seems relatively large. If this Pull Request contains multiple changes, please split each into separate PR will helps faster, easier review.");
 }
+
```

Comment:
Ditch the comment? Are you done with this PR?

Reply:
nope, not done